src/main/java/com/booksaw/betterTeams/TeamPlayer.java

```diff
@@ -81,6 +81,10 @@ public TeamPlayer(@NotNull String data) {
 		teamChat = false;
 	}
 
+	public UUID getID() {
```

Comment:
did this previously exist and was removed prior or something?

Reply:
No, as far as I know it didn't.